app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -55,7 +55,12 @@ interface Pixel {
         SETTINGS_THEME_TOGGLED_DARK("ms_td"),
 
         HTTPS_UPGRADE_SITE_ERROR("ehd"),
-        HTTPS_UPGRADE_SITE_SUMMARY("ehs")
+        HTTPS_UPGRADE_SITE_SUMMARY("ehs"),
+
+        SURVEY_CTA_SHOWN(pixelName = "mus_cs"),
```

Comment:
has the server-side work been done for the pixels?

Reply:
Yes, done and merged.